### PR TITLE
Upgrade: use oceanifier@4.0.1 to simplify test/runner.html

### PR DESCRIPTION
**Pull request by cyjake**
I've refactored and enhanced oceanifier a lot. With oceanifier@4.0.1, the `test/runner.html` can be simplified drastically.

### CODE REVIEW

File: test/runner.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 'use strict'
 
+mocha.setup('bdd')
+
 require('./test.heredoc')
 require('./test.heredoc.strip')
+
+mocha.run()
```
Comment (dotnil) [inline test/runner.js]: :flushed: didn't test this in Node. I'll see to it asap.

File: test/runner.html [hunk 2]
```diff
@@ -8,16 +8,6 @@
...
-  <script src="http://amo.alicdn.com/L1/377/101010/sea.js"></script>
-  <script src="/config.js"></script>
-  <script>
-    /* eslint-disable */
-    mocha.ui('bdd')
-    define('should', should)  // use the pre-build version of should
-    seajs.use('heredoc/test/runner', function() {
-      mocha.run()
-    })
-  </script>
```
Comment (dotnil) [inline test/runner.html]: This PR is about this change mostly. There's no more strange stuff here. Hope you like it @jden

File: test/runner.js [hunk 3]
```diff
@@ -1,4 +1,9 @@
 'use strict'
+/* globals mocha */
+
+if (typeof mocha !== 'undefined') mocha.setup('bdd')
 
 require('./test.heredoc')
 require('./test.heredoc.strip')
+
+if (typeof mocha !== 'undefined') mocha.run()
```
Comment (dotnil) [inline test/runner.js]: When running in browser, we need to perform `mocha.setup()` and `mocha.run()`,

**Comment by dotnil** [comment]
ping @jden

**Comment by junosuarez** [comment]
sorry for the unusually long delay! I've been travelling without regular internet access ^.^

merged. since this is a dev-only change, I haven't republished on npm.

**Pull request merged**